src/lib/db/migrations/00027-admin-community.js

```
@@ -0,0 +1,62 @@
+/** @param {import('postgres').Sql<any>} sql */
+export const up = async (sql) => {
+	// Defer the `membership.persona_id` foreign key constraint to the end of the transaction:
+	await sql`
+		ALTER TABLE memberships
+		DROP CONSTRAINT memberships_persona_id_fkey,
+		ADD CONSTRAINT memberships_persona_id_fkey
+			FOREIGN KEY (persona_id)
+			REFERENCES personas (persona_id)
+			ON UPDATE CASCADE ON DELETE CASCADE
+			INITIALLY DEFERRED;
+	`;
```

Seed was failing when this PR was combined with the transaction PR. The added bit is `INITIALLY DEFERRED` -- it allows the constraint to be checked only at the end of the transaction. More here: https://www.postgresql.org/docs/current/sql-createtable.html

---

Hmm adding this makes the membership silently fail -- I think if seed ran in a single transaction, it would have caught this error.

I simply removed the constraint and the membership succeeds. Going to poke around a bit...

---

It's now seeding with a single transaction, so if anything goes wrong nothing gets seeded, doesn't solve our problem though.

---

Figured out the problem! `serviceRequest.repos` isn't using the transacting `repos`

# 🤦

---

Ah, yup, that'll do it. Great job figuring that out though Ryan!